providers/gce/gce_loadbalancer.go

```diff
@@ -133,7 +133,7 @@ func (g *Cloud) EnsureLoadBalancer(ctx context.Context, clusterName string, svc
 	// GCE load balancers do not support services with LoadBalancerClass set. LoadBalancerClass can't be updated for an existing load balancer, so here we don't need to clean any resources.
 	// Check API documentation for .Spec.LoadBalancerClass for details on when this field is allowed to be changed.
 	if svc.Spec.LoadBalancerClass != nil {
-		klog.Infof("Ignoring service %s/%s using load balancer class %s, it is not supported by this controller.", svc.Namespace, svc.Name, svc.Spec.LoadBalancerClass)
+		klog.Infof("Ignoring service %s/%s using load balancer class %s, it is not supported by this controller.", svc.Namespace, svc.Name, *svc.Spec.LoadBalancerClass)
```

Comment:
This looks like the only consequential change. Did we manually test it? Should we add unit/e2e tests?

Reply:
It doesn't pass govet if you don't add this patch...